### ISSUE TITLE
Printing a non-existent file gives a weird error

### ISSUE THREAD

**Issue by grosser** [sparkleformation/sparkle_formation#116]
@chrisroberts not urgent, just annoying / confusing

```
sfn print -f does_not_exist.rb
ERROR: NoMethodError: undefined method `push' for nil:NilClass
```

**Comment by chrisroberts** [comment]
I cannot reproduce this error. Perhaps it's a configuration issue or bad content. `DEBUG=true` will provide the stacktrace and will be helpful to determine the origin and provide a useful error message.

```
→ sfn print -f does_not_exist.rb
ERROR: SparkleFormation::Error::NotFound::Template: ["No template registered with requested name (does_not_exist.rb)!"]
```

**Comment by grosser** [comment]
looks like it needs at least 1 component, see https://github.com/grosser/sparkle-load/tree/test

**Comment by luckymike** [comment]
I just checked out the repo and attempted to reproduce:
```
~/Development/sparkle-load (master) $ sfn print -f not-here.rb
TESTING
ERROR: SparkleFormation::Error::NotFound::Template: ["No template registered with requested name (not-here.rb)!"]
```
And running the exact example: 
```
~/Development/sparkle-load (master) $ sfn print -f does_not_exist.rb
TESTING
ERROR: SparkleFormation::Error::NotFound::Template: ["No template registered with requested name (does_not_exist.rb)!"]
```

**Comment by chrisroberts** [comment]
It is reproducible on the `test` branch. I'll leave this issue open so that I include an init guard within the pack. However, the contents of the repository are not structured or implemented in a manner defined within the documentation and as such the behavior around this usage is and should be considered undefined.

**Comment by luckymike** [comment]
My bad, I missed the branch when cloning.

**Comment by chrisroberts** [comment]
This was fixed by #123

**Comment by grosser** [comment]
:heart: thx for following up on that :D
